test/TritonGPU/amd/accelerate-amd-matmul-mfma.mlir

```diff
@@ -45,6 +45,41 @@ module attributes {"ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 8 : i32, ttg.targ
   }
 }
 
+// -----
+
+// MFMA0: amd_mfma
+// MFMA16: amd_mfma
+// CHECK-LABEL: small_m_size_mfma
+#blocked = #ttg.blocked<{sizePerThread = [4, 4], threadsPerWarp = [1, 64], warpsPerCTA = [1, 2], order = [1, 0], CTAsPerCGA = [1, 1], CTASplitNum = [1, 1], CTAOrder = [1, 0]}>
+module attributes {"ttg.target" = "hip:gfx942", "ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 2 : i32, "ttg.threads-per-warp" = 64 : i32} {
+  tt.func @small_m_size_mfma(
+    %a: tensor<4x64xf16, #ttg.dot_op<{opIdx = 0, parent = #blocked}>>,
+    %b: tensor<64x128xf16, #ttg.dot_op<{opIdx = 1, parent = #blocked}>>)
+    -> tensor<4x128xf32, #blocked> {
+    %zero_f32 = arith.constant dense<0.000000e+00> : tensor<4x128xf32, #blocked>
+    %result = tt.dot %a, %b, %zero_f32 : tensor<4x64xf16, #ttg.dot_op<{opIdx = 0, parent = #blocked}>> * tensor<64x128xf16, #ttg.dot_op<{opIdx = 1, parent = #blocked}>> -> tensor<4x128xf32, #blocked>
+    tt.return %result : tensor<4x128xf32, #blocked>
+  }
+}
+
+// -----
+
+// MFMA0-NOT: amd_mfma
+// MFMA16: amd_mfma
```

Comment:
Similarly here good to check more details including M/N size.

Reply:
done